geo/src/algorithm/concavehull.rs

```diff
@@ -0,0 +1,418 @@
+use crate::algorithm::convexhull::ConvexHull;
```

Comment:
one more nit: could you rename this `concave_hull.rs` to be in line with the rest of the crate?